util/testplanner/class_defs.py

```diff
@@ -322,9 +324,12 @@ def results_table(self, regr_results, map_full_testplan=True, fmt="pipe"):
                     milestone, entry_name, test["name"], test["passing"],
                     test["total"], pass_rate
                 ])
+                if entry == self.entries[-1]:
+                    self.final_total = [
+                        self.name.upper(), test["passing"], test["total"],
+                        pass_rate
+                    ]
                 milestone = ""
                 entry_name = ""
-        return tabulate(table,
-                        headers="firstrow",
-                        tablefmt="pipe",
-                        colalign=colalign)
+        self.results = tabulate(table, headers="firstrow", tablefmt="pipe")
```

Comment:
colalign arg accidentally dropped?

Reply:
My mistake. it was dropped by manual merge.